runelite-client/pom.xml

```diff
@@ -38,18 +38,6 @@
 	<properties>
 		<guice.version>4.1.0</guice.version>
 		<jogl.version>2.3.2</jogl.version>
-		<hamcrest.version>1.3</hamcrest.version>
```

Comment:
This versioning scheme seemed good, was there an issue with it I'm unaware of?

Reply:
Just personal preference I guess